src/Sarif/Readers/FlagsEnumConverter.cs

```diff
@@ -53,7 +53,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 throw new ArgumentNullException(nameof(writer));
             }
 
-            if (!(value is SuppressionStates || value is ArtifactRoles))
```

Comment:
>SuppressionStates [](start = 27, length = 17)

i removed it since the SuppressionKind is not a flagsEnum.

Reply:
thought: is it possible to make this check to be more generic so that we dont have to check the type (for future flags)?

---
In reply to: [268348737](https://github.com/Microsoft/sarif-sdk/pull/1361#discussion_r268348737) [](ancestors = 268348737)